TestAssets/TestProjects/StandaloneApp/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc3-*",
+        "Newtonsoft.Json": "9.0.1-beta1"
```

Comment:
See explanation on the portable app, it's necessary for things like --additionalprobingpath validation. I don't think we should have an explosion of testassets either because that has perf issues.

---
In reply to: [66346600](https://github.com/dotnet/core-setup/pull/97#discussion_r66346600) [](ancestors = 66346600)